packages/astro/test/astro-global-prerender.test.js

```diff
@@ -0,0 +1,91 @@
+// @ts-check
```

Comment:
We already have an `astro-global.test.js`. what's the reason for creating a new file?

Reply:
That's because the new property relies on the config output. I could not manipulate the fixture as i wanted in the file you mention